README.md

```diff
@@ -15,7 +15,7 @@ Get to know the Fastly Compute@Edge environment with a basic starter that demons
 
 ## Understanding the code
 
-This starter is intentionally lightweight, and requires no dependencies aside from the [`@fastly/as-compute`](https://npmjs.com/package/@fastly/as-compute) package. It will help you understand the basics of processing requests at the edge using Fastly. This starter includes implementations of common patterns explained in our [using Compute@Edge](/learning/compute/using/) and [VCL migration](/learning/compute/migrate) guides.
+This starter is intentionally lightweight, and requires no dependencies aside from the [`@fastly/as-compute`](https://npmjs.com/package/@fastly/as-compute) package. It will help you understand the basics of processing requests at the edge using Fastly. This starter includes implementations of common patterns explained in our [using Compute@Edge](/learning/compute/assemblyscript/) and [VCL migration](/learning/compute/migrate/) guides.
```

Comment:
Should the link text reflect the actual title rather than being the same as the original page content?

i.e. "AssemblyScript on Compute@Edge"

Reply:
It's consistent with the Rust one.  I think it's OK.